guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -1,162 +1,40 @@
 [id="upgrading_a_connected_{project-context}_server_{context}"]
 = Upgrading a connected {ProjectServer}
 
-Use this procedure for a {ProjectServer} with access to the public internet
+Upgrading a {ProjectServer} with access to the public internet includes backing up the server, ensuring that the system and {Project} configuration is up-to-date, and running the upgrade.
 
 include::snip_warning-maintain-config-noop.adoc[]
 
-include::snip_prerequisites-upgrading-project-server.adoc[]
-
-.Upgrade {ProjectServer}
-
-. Stop all {Project} services:
-+
-[options="nowrap" subs="+quotes,verbatim,attributes"]
-----
-# {foreman-maintain} service stop
-----
-. Take a snapshot or create a backup:
-* On a virtual machine, take a snapshot.
-* On a physical machine, create a backup.
-+
-. Start all {Project} services:
-+
-[options="nowrap" subs="+quotes,verbatim,attributes"]
-----
-# {foreman-maintain} service start
-----
-
-. Optional: If you made manual edits to DNS or DHCP configuration in the `/etc/zones.conf` or `/etc/dhcp/dhcpd.conf` files, back up the configuration files because the installer only supports one domain or subnet, and therefore restoring changes from these backups might be required.
-
-. Optional: If you made manual edits to DNS or DHCP configuration files and do not want to overwrite the changes, enter the following command:
+.Prerequisites
+* If you have made manual edits to you DNS or DHCP configuration in the `/etc/zones.conf` or `/etc/dhcp/dhcpd.conf` files, ensure DNS and DHCP configuration management is disabled:
```

Comment:
Reading this, it now makes me wonder: Is there a risk that some users will not remember whether they made manual edits so they'll disable this "just in case"? Should we warn them not to do it unless absolutely required? Or am I just underestimating our users?

Reply:
In 98% of the cases I hope they disabled management when they started changing those files (otherwise they would have been overwritten already), so you probably could even get away with dropping this line altogether.

But no, I don't think people will do that "just in case"